packages/core/src/extension/extension-commands.ts

```diff
@@ -4,6 +4,10 @@ export type ExtensionCommand =
     | SetIframeStyleCommand
     | GetConsignmentsCommand;
 
+export enum InstantDataCommandType {
+    Consignments = 'consignments',
+}
+
```

Comment:
Introducing a new set of data commands that work on top of the existing command and message types, allowing developers to write `extensionService.get('consignments')` to fetch data.